src/odemis/cli/convert.py

```diff
@@ -210,12 +210,13 @@ def add_acq_type_md(das):
     return das
 
 
-def stitch(infns, registration_method, weaving_method):
+def stitch(infns: list, registration_method: int = REGISTER_IDENTITY, weaving_method: int = WEAVER_MEAN) -> list:
```

Comment:
Please move this function to `odemis.util.dataio` as something like `open_and_stitch()`